src/Template/Imagemap/ImagemapTemplate.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace LineMob\Core\Template\Imagemap;
+
+use LINE\LINEBot\ImagemapActionBuilder\ImagemapMessageActionBuilder;
+use LINE\LINEBot\ImagemapActionBuilder\ImagemapUriActionBuilder;
+use LINE\LINEBot\MessageBuilder\Imagemap\BaseSizeBuilder;
+use LINE\LINEBot\MessageBuilder\ImagemapMessageBuilder;
+use LineMob\Core\Template\AbstractTemplate;
+
+class ImagemapTemplate extends AbstractTemplate
+{
+    /**
+     * @var string
+     */
+    public $baseUrl;
+
+    /**
+     * @var string
+     */
+    public $altText = 'this is an imagemap';
+
+    /**
+     * @var int
+     */
+    public $width;
+
+    /**
+     * @var int
+     */
+    public $height;
+
+    /**
+     * @var Action[]
+     */
+    public $actions;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplate()
+    {
+        $baseSize = new BaseSizeBuilder($this->height, $this->width);
+        $actions = [];
+
+        foreach ($this->actions as $action) {
+            switch ($action->type) {
+                case Action::TYPE_URI;
+                    $actions[] = new ImagemapUriActionBuilder($action->link, $action->area->getArea());
+                    break;
+
+                case Action::TYPE_MESSAGE;
+                    $actions[] = new ImagemapMessageActionBuilder($action->text, $action->area->getArea());
+                    break;
+            }
+        }
```

Comment:
@liverbool 
I think `addAction` don't need anymore because already have `addLinkAction` and `addMessageAction`. It's already separate.

What do you think ? Should I remove `addAction` ?